components/financial_assistance/spec/controllers/financial_assistance/verification_documents_controller_spec.rb

```diff
@@ -242,12 +304,23 @@
   end
 
   context 'broker logged in' do
```

Comment:
can you also include two more example related to 
- when authorized broker_staff_user logged in try to access the pages.
- when unauthorized broker_staff_user logged in try to access the pages.

Reply:
https://github.com/ideacrew/enroll/pull/3693/commits/9d2dec224cd2d6793b23aef18de83cbe8063b193#diff-e31e73ba48af1ac2f7034a00c87949721773ae9e56b1a2559ac64cd5d3790ff3R462